README.md

```diff
@@ -4,20 +4,47 @@
 
 The [Draft U.S. Web Design Standards](https://standards.usa.gov) include a library of open source UI components and a visual style guide for U.S. federal government websites.
 
-These tools follow industry-standard web accessibility guidelines and use the best practices of existing style libraries and modern web design. Created and maintained by [U.S. Digital Service](https://www.whitehouse.gov/digital/united-states-digital-service) and [18F](https://18f.gsa.gov/) designers and developers, the Draft Web Design Standards are designed for use by government product teams who want to create beautiful, easy-to-use online experiences for the public. To learn more about the project, check out this [blog post](https://18f.gsa.gov/2015/09/28/web-design-standards/).
+Previously, the website and documentation for the Draft U.S. Web Design Standards were also part of this repository. To provide more clarity to both people who want to work with the standards and people who work with the documentation locally, we have separated these into two repositories. This repository is for the Standards themselves.
+
+18F maintains [another repository for the documentation and website](https://github.com/18F/web-design-standards-docs). To see the standards and documentation on the web, visit [https://standards.usa.gov](https://standards.usa.gov).
+
+## Contents
+
+* [Background](#background)
+* [Recent updates](#recent-updates)
+* [Getting started](#getting-started)
+* [Use the Standards](#using-the-standards)
+  * [Download](#download)
+  * [Install using npm](#install-using-npm)
+  * [Use another framework or package manager](#use-another-framework-or-package-manager)
+* [Our use of branches](#our-use-of-branches)
+* [Need installation help?](#need-installation-help)
+* [Contributing to the code base](#contributing-to-the-codebase)
+* [Reuse of open-source style guides](#reuse-of-open-source-style-guides)
+* [Licenses and attribution](#licenses-and-attribution)
+
+## Background
+
+The components and style guide of the Draft U.S. Web Design Standards follow industry-standard web accessibility guidelines and use the best practices of existing style libraries and modern web design. The [U.S. Digital Service](https://www.whitehouse.gov/digital/united-states-digital-service) and [18F](https://18f.gsa.gov/) created and maintain the Draft Web Design Standards for designers and developers. They are designed for use by government product teams who want to create beautiful, easy-to-use online experiences for the public. To learn more about the project, check out this [blog post](https://18f.gsa.gov/2015/09/28/web-design-standards/).
```

Comment:
@juliaelman To respond to your point, I either use the full name ("Draft U.S. Web Design Standards") or simply "Standards" — @jameshupp thoughts?